src/main/scala/sigmastate/types.scala

```diff
@@ -427,6 +427,7 @@ case object SSigmaProp extends SProduct with SPrimType with SEmbeddable with SLo
           GroupElementConstant(gv), GroupElementConstant(hv),
           GroupElementConstant(uv), GroupElementConstant(vv)) =>
       SGroupElement.dataSize(gv.asWrappedType) * 4 + 1
+    case CAND(inputs) => inputs.map(i => dataSize(i.asWrappedType)).sum
```

Comment:
@ergomorphic Could you please check my costing for `CAND`?

Reply:
@greenhat looks good.